### PR TITLE
88 set up of testnet env with single click

### PR DESCRIPTION
**Pull request by AntonAndell**
Beginning of a integration testing framework running on a local node.

### CODE REVIEW

**Review by nightowl121 — CHANGES_REQUESTED** [review]

File: .gitignore [hunk 1]
```diff
@@ -18,7 +18,7 @@
 *.zip
 *.tar.gz
 *.rar
-
+!integrationtest-lib/python-contracts/*.zip
```
[the remark this answers is not in the document]
Comment (AntonAndell) [reply .gitignore:21]: Lol goddam, Move the files into the testlib and forgot the git ignore. Im no home until tonight. If you checkout the first commit they should be there. If you want you can add them and push them up

File: test-lib/src/main/java/network/balanced/score/lib/test/integration/Balanced.java [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+
+    protected void setupContracts() {
+        stakingScore = new StakingScoreClient(staking);
+
+        stakingScore.setSicxAddress(sicx._address());
+        governanceScore.configureBalanced();
+        governanceScore.launchBalanced();
+        stakingScore.toggleStakingOn();
+    }
+}
```
[the remark this answers is not in the document]
Comment (AntonAndell) [reply test-lib/src/main/java/network/balanced/score/lib/test/integration/Balanced.java:132]: Im thinking of splitting the balanced methods a lot more. So you can deconstruct it however you want to. But thinking we will do this when we are hands on with testing. Will be much easier to find the correct solutions then

File: .gitignore (hunk 1 above)
[the remark this answers is not in the document]
Comment (AntonAndell) [reply .gitignore:21]: nvm they aren't there either

File: test-lib/src/main/java/network/balanced/score/lib/test/integration/Balanced.java [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+
+    protected void deployContracts() {
+        baln = deploy(owner, "Baln", Map.of("_governance", governance._address()));
+        bwt = deploy(owner, "Bwt", Map.of("_governance", governance._address()));
+        dex = deploy(owner, "Dex", Map.of("_governance", governance._address()));
+        feehandler = deploy(owner, "Feehandler", Map.of("_governance", governance._address()));
+        loans = deploy(owner, "Loans", Map.of("_governance", governance._address()));
+        rebalancing = deploy(owner, "Rebalancing", Map.of("_governance", governance._address()));
+        rewards = deploy(owner, "Rewards", Map.of("_governance", governance._address()));
+        sicx = deploy(owner, "Sicx", Map.of("_admin", governance._address()));
```
Comment (Suyog007) [inline test-lib/src/main/java/network/balanced/score/lib/test/integration/Balanced.java:93]: admin of sicx will be staking address not governance

**Review by nightowl121 — APPROVED** [review]

**Pull request merged**